### PR TITLE
Bump citizens_advice_form_builder to 0.1.4

### PR DESCRIPTION
**Pull request by leeky**
Changelog:

- Add error handling to `cads_date_field`
- Fix release instructions

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ Enter `gem install gem-release` to install on your local machine.
 
 ## Release Process
 
+- Use `gem bump -v patch` to bump the patch release number of the gem
 - Update `CHANGELOG.md` with any changes going into this release
-- Use `gem bump -v minor` to bump the minor release number of the gem
```
Comment (marianayovcheva) [inline RELEASE.md:14]: Curiosity question - why is this line removed?

**Review by marianayovcheva — APPROVED** [review]
LGTM! 🚀

File: RELEASE.md (hunk 1 above)
Comment (leeky) [reply RELEASE.md:14]: It's still there, just the order of the instructions was wrong.

Comment (marianayovcheva) [reply RELEASE.md:14]: Sorry, I meant why is `minor` replaced by `patch`?

Comment (leeky) [reply RELEASE.md:14]: Because so far I've only been releasing patch versions (i.e. `0.1.2`, `0.1.3`, `0.1.4`). The next minor version would be `0.2.0`.

Once it's reasonably stable, I'll bump the version to v1.0, and then amend the release notes to increment the minor version from then on.

Comment (marianayovcheva) [reply RELEASE.md:14]: Makes sense, thanks!
